rcl/src/rcl/node.c

```diff
@@ -52,6 +52,8 @@ extern "C"
 
 #include "./context_impl.h"
 
+const char * const RCL_DISABLE_LOAN_MSG_ENV_VAR = "ROS_DISABLE_LOAN_MSG";
```

Comment:
I think `ROS_DISABLE_LOANED_MESSAGES` is a better variable name (everywhere, including the env var and the code).